rustls/src/keylogfile.rs

```diff
@@ -0,0 +1,151 @@
+use crate::KeyLog;
```

Comment:
I think the file should probably be called `keylog_file` or (better, for consistency) `key_log_file`.